api/app/src/main/resources/errorBundle.properties

```diff
@@ -35,6 +35,8 @@ noRolesToUpdateWithPermission=No roles to update with permission
 notAllFilesFound=One or more file permissions not found
 packetGroupNotFound=Packet group not found
 packetNotFound=Packet not found
+pinAlreadyExists=Pin already exists
```

Comment:
pinned packet? maybe reference to packet?

Reply:
Changing to 'packet already pinned'